monai/handlers/checkpoint_saver.py

```diff
@@ -116,7 +116,8 @@ class _DiskSaver(DiskSaver):
             """
 
             def __init__(self, dirname: str, filename: Optional[str] = None):
-                super().__init__(dirname=dirname, require_empty=False)
+                # set `atomic=False` as `atomic=True` may save files without read permission for non-root users
+                super().__init__(dirname=dirname, require_empty=False, atomic=True)
```

Comment:
Comment (`atomic=False`) doesn't seem to correspond to code (`atomic=True`)?

Reply:
Sorry for my typo, I changed the code to double confirm and forgot to change back...